packages/atlas/src/components/_comments/Comment/Comment.styles.ts

```diff
@@ -0,0 +1,70 @@
+import styled from '@emotion/styled'
+
+import { Text } from '@/components/Text'
+import { IconButton } from '@/components/_buttons/IconButton'
+import { SvgActionTrash } from '@/components/_icons'
+import { cVar, sizes } from '@/styles'
+
+export const KebabMenuIconButton = styled(IconButton)<{ isActive: boolean }>`
+  opacity: 0;
+  pointer-events: ${({ isActive }) => (isActive ? 'auto' : 'none')};
+
+  @media (any-pointer: coarse) {
+    opacity: ${({ isActive }) => (isActive ? 1 : 0)};
+  }
+`
+
+export const CommentWrapper = styled.div<{ shouldShowKebabButton: boolean }>`
+  display: grid;
+  gap: ${sizes(3)};
+  align-items: start;
+
+  /* comment content, kebab button */
+  grid-template-columns: 1fr auto;
+
+  :hover {
+    ${KebabMenuIconButton} {
+      opacity: ${({ shouldShowKebabButton: isActive }) => (isActive ? 1 : 0)};
+    }
+  }
+`
+
+export const CommentContent = styled.div`
+  margin-bottom: ${sizes(2)};
+`
+
+export const CommentHeader = styled.div`
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  margin-bottom: ${sizes(2)};
+`
+
+export const CommentHeaderDot = styled.div`
+  background-color: ${cVar('colorBackgroundAlpha')};
+  border-radius: 100%;
+  width: ${sizes(1)};
```

Comment:
if I remember correctly we decided to not use `sizes` for width and height. Also, just a reminder that we have `square` helper 😛